### ISSUE TITLE
Incorrect display of Classification value on saved Provisional panel

### ISSUE THREAD

**Issue by selinad** [ClinGen/clincoded#1668]
A user wrote in that, for a Definitive gene where no modification was provided, the saved Provisional panel looked like this (below), but the View Summary had the correct information.  

![image](https://user-images.githubusercontent.com/11320314/39316243-78276e32-492d-11e8-9e84-74f9dd800465.png)

**Comment by jimmyzhen** [comment]
Hi @selinad, @wrightmw,

You may review the fix in the following instance:
https://jz-experimental-checkbox.demo.clinicalgenome.org

**Comment by selinad** [comment]
Looks perfect now!

**Comment by wrightmw** [comment]
Yup. Can see the Classification on the saved Provisional now even when there is no Modification.
